nixie/cli.py

```
@@ -212,14 +212,12 @@ async def render_block(
         cli = exc.filename or "mmdc"
         print(
-            "Error: '{0}' not found. Install Node.js with npx or Bun to use @mermaid-js/mermaid-cli.".format(
-                cli
-            ),
+            f"Error: '{cli}' not found. Install Node.js with npx or Bun to use @mermaid-js/mermaid-cli.",
             file=sys.stderr,
         )
     except RuntimeError as exc:
         print(exc, file=sys.stderr)
-    except Exception as exc:
+    except Exception as exc:  # pragma: no cover - unexpected
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)